client/verifier.go

```diff
@@ -46,17 +47,21 @@ func (c *logVerifier) VerifyRoot(trusted, newRoot *trillian.SignedLogRoot,
 	consistency [][]byte) error {
 
 	// Verify SignedLogRoot signature.
-	hash := tcrypto.HashLogRoot(*newRoot)
-	if err := tcrypto.Verify(c.pubKey, hash, newRoot.Signature); err != nil {
-		return err
+	if (newRoot == nil){
+		return fmt.Errorf("VerifyRoot() error: newRoot == nil.")
```

Comment:
Actually, sorry for the back and forth here, but since the capitalized string is actually the method name, what you had originally is correct.

We don't capitalize sentences in errors because error chaining is a common practice, so it would look weird, but in this case it works fine.

I'd go with a small change, which is dropping the " error" part: `fmt.Errorf("VerifyRoot(): newRoot == nil")`